docs/_includes/Statements.md

```diff
@@ -99,15 +139,18 @@ These expressions are then evaluated, then they are
 assigned to the yield parameters, and then the iterator
 yields.
 
-## Update and Call Statements
+## Update and Call Statements {#sec-update-and-call-statement}
 ````
 UpdateStmt =
     Lhs
     ( {Attribute} ";"
```

Comment:
The grammar explained here is very concrete. This makes the two cases where `Lhs` denote a call (L146 and L162) confusing. Can this be changed in the reference manual to make the grammar more abstract?